src/screens/index.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Home from './Home';
+import About from './About';
+import Faq from './Faq';
+import ContactUs from './ContactUs';
+
+const MainBody = (props) => {
+    const {screen} = props;
+    let returnScreen;
+    switch (screen) {
+        case 'About':
+            returnScreen = <About {...props} />;
+            break;
+        
+    case 'Faq':
```

Comment:
The indentation in this file is a bit inconsistent. Could you use `prettier` as described in my other comment?